openstack/setup.py

```diff
@@ -62,7 +77,7 @@ def get_dependencies():
     packages=['datadog_checks.openstack'],
     # Run-time dependencies
     install_requires=[CHECKS_BASE_REQ],
-    extras_require={'deps': get_dependencies()},
```

Comment:
I just realized, do we still need the `get_dependencies()` method in this file? It's still defined in the [setup.py](https://github.com/DataDog/integrations-core/pull/11408/files#diff-ba25647a7e8cf4fc0715da9ea2dd936860440703854e0b33d3bdc836f317238bR24-R30) for this check and I think the other backend build PRs.

Reply:
Ah never mind, the `setup.py` file is going to be removed in the near future anyway.